script/tool/lib/src/create_all_packages_app_command.dart

```diff
@@ -98,80 +114,152 @@ class CreateAllPackagesAppCommand extends PackageCommand {
   }
 
   Future<int> _createApp() async {
-    final io.ProcessResult result = io.Process.runSync(
+    return processRunner.runAndStream(
```

Comment:
Is this modifying the current all packages test to use kotlin?

Reply:
Yes; this is mentioned in the PR description. I'm not sure why it wasn't already, given that it's been the default for a long time.

Reply:
yeah I read that in the description and was looking for the place where we made the swap. Where in the cl do we test the legacy version with java?